ipv8/test/messaging/anonymization/test_hiddenservices.py

```diff
@@ -28,6 +27,10 @@ def announce(self, info_hash):
             global_dht_services[info_hash] = [self.address]
 
 
+class Twisted(object):
```

Comment:
Why do we need this?

Reply:
the reason for this is too much alt+enter smashing in PyCharm...
Funny, though, that pylint don't catch it.

Reply:
We run a quick version of Pylint. The older version we run doesn't catch all of the errors, it does finish in 5 seconds instead of 2 minutes though.